src/Dapr.AspNetCore/ITopicMetadata.cs

```diff
@@ -28,6 +28,11 @@ public interface ITopicMetadata
         /// </summary>
         string PubsubName { get; }
 
+        /// <summary>
+        /// Gets the dead letter topic.
+        /// </summary>
+        string DeadLetterTopic { get; }
```

Comment:
This is a breaking change: https://docs.microsoft.com/en-us/dotnet/core/compatibility/#members

The way we've done this in the past is usually adding a new interface that specifies what we're looking for. e.g. `IDeadLetterTopicMetadata`.

Truthfully though, I haven't really liked this pattern and I don't think it fits in with .NET's own recommendation for when to use interfaces. I'd try just adding the attribute directly to the `TopicAttribute` and see if that works. My only concern here is that the interfaces might be used for the encoding/decoding path.

Reply:
Fixed !